mappings/net/minecraft/server/network/ServerLoginNetworkHandler.mapping

```diff
@@ -16,7 +16,7 @@ CLASS net/minecraft/unmapped/C_rktsiyfy net/minecraft/server/network/ServerLogin
 	FIELD f_dosbbvgm loginTicks I
 	FIELD f_jtougkjq RANDOM Lnet/minecraft/unmapped/C_rlomrsco;
 	FIELD f_kankoaiw serverId Ljava/lang/String;
-	FIELD f_krozceue NEXT_AUTHENTICATOR_THREAD_ID Ljava/util/concurrent/atomic/AtomicInteger;
+	FIELD f_krozceue nextAuthenticatorThreadId Ljava/util/concurrent/atomic/AtomicInteger;
```

Comment:
We use upper snake-case for atomics iirc?

Reply:
`FieldNamingChecker`, line 27-31
```
        if (isAtomic) {
            if (startsWithUppercase(mapping.targetName())) {
                errorReporter.warning("atomic field starts with uppercase character '" + mapping.targetName().charAt(0) + "'");
            }
        }
```

Reply:
Yeah i think this should move away from SNAKE_CASE